src/shared/dropdown-list-v2/dropdown-list.tsx

```diff
@@ -84,6 +89,15 @@ export const DropdownList = <T, V>({
     const searchInputRef = useRef<HTMLInputElement>(null);
     const virtuosoRef = useRef<VirtuosoHandle>(null);
 
+    const hasSelectedMax = useMemo(
```

Comment:
just curious why the need for useMemo as it looks like a pretty straightforward computation